File: swagger.jersey.demo/src/main/java/com/tobiascode/Movie.java
package com.tobiascode;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

@ApiModel( value = "Movie", description = "All the details about a movie" )
public class Movie {
	@ApiModelProperty( value = "Movie's id")private int id;
	@ApiModelProperty( value = "Title of the movie")private String title;
	@ApiModelProperty( value = "Who directed the movie")private String director;
	@ApiModelProperty( value = "Short description about the movie")private String synopsis;
	@ApiModelProperty( value = "Year it was released")private String yearOfRelease;

	public Movie() {
	}

	public Movie(int id, String title, String director, String synopsis,
			String yearOfRelease) {
		setId(id);
		setTitle(title);
		setDirector(director);
		setSynopsis(synopsis);
		setYearOfRelease(yearOfRelease);
	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getTitle() {
		return title;
	}

	public void setTitle(String title) {
		this.title = title;
	}

	public String getDirector() {
		return director;
	}

	public void setDirector(String director) {
		this.director = director;
	}

	public String getSynopsis() {
		return synopsis;
	}

	public void setSynopsis(String synopsis) {
		this.synopsis = synopsis;
	}

	public String getYearOfRelease() {
		return yearOfRelease;
	}

	public void setYearOfRelease(String yearOfRelease) {
		this.yearOfRelease = yearOfRelease;
	}
}

File: clients/csharp/src/main/csharp/io/swagger/Model/Movie.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// All the details about a movie
  /// </summary>
  [DataContract]
  public class Movie {
    
    /* Movie's id */
    [DataMember(Name="id", EmitDefaultValue=false)]
    public int? Id { get; set; }

    
    /* Title of the movie */
    [DataMember(Name="title", EmitDefaultValue=false)]
    public string Title { get; set; }

    
    /* Who directed the movie */
    [DataMember(Name="director", EmitDefaultValue=false)]
    public string Director { get; set; }

    
    /* Short description about the movie */
    [DataMember(Name="synopsis", EmitDefaultValue=false)]
    public string Synopsis { get; set; }

    
    /* Year it was released */
    [DataMember(Name="yearOfRelease", EmitDefaultValue=false)]
    public string YearOfRelease { get; set; }

    

    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Movie {\n");
      
      sb.Append("  Id: ").Append(Id).Append("\n");
      
      sb.Append("  Title: ").Append(Title).Append("\n");
      
      sb.Append("  Director: ").Append(Director).Append("\n");
      
      sb.Append("  Synopsis: ").Append(Synopsis).Append("\n");
      
      sb.Append("  YearOfRelease: ").Append(YearOfRelease).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
  
  
}